<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Госзакупки 0.2.2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>%p%  (%v/%m)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="verticalTabWidget_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>432</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="verticalTabWidget_2Page1">
       <attribute name="title">
        <string>Поиск по UID</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите номер-идентификатор:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="text">
             <string>2623417307018000013</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Искать</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout" name="horizontalLayout_4"/>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Документы</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox">
                <property name="text">
                 <string>Автоматически загружать документы</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Поиск по списку ID</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Очистить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Экспорт в xls</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>            Имя файла:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>.xlsx</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Поиск на zakupki.gov.ru</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>571</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPushButton" name="pushButton_10">
           <property name="text">
            <string>Скачать список организаций</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>240</y>
          <width>244</width>
          <height>72</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>68</width>
              <height>68</height>
             </rect>
            </property>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>131</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>ИНН получателя</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>10</y>
               <width>161</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>3711013676</string>
             </property>
            </widget>
            <widget class="QPlainTextEdit" name="plainTextEdit_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>200</y>
               <width>1167</width>
               <height>101</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>131</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>Период времени</string>
             </property>
            </widget>
            <widget class="QDateEdit" name="dateEdit">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>40</y>
               <width>111</width>
               <height>21</height>
              </rect>
             </property>
            </widget>
            <widget class="QDateEdit" name="dateEdit_2">
             <property name="geometry">
              <rect>
               <x>360</x>
               <y>40</y>
               <width>110</width>
               <height>21</height>
              </rect>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBox_2">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>40</y>
               <width>41</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>c</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBox_3">
             <property name="geometry">
              <rect>
               <x>310</x>
               <y>40</y>
               <width>41</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>по</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Искать</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Очистить</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>170</y>
          <width>291</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7"/>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>120</y>
          <width>291</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5"/>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Поиск по budget.gov.ru</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>631</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>Обновить базы</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>Загрузить базу в память</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>601</width>
          <height>281</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>599</width>
           <height>279</height>
          </rect>
         </property>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>10</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>3711013676</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Период времени</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_4">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>40</y>
            <width>41</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>c</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>ГБРС ИНН</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_5">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>40</y>
            <width>41</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>по</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="dateEdit_3">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>40</y>
            <width>110</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QDateEdit" name="dateEdit_4">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>40</y>
            <width>111</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>10</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>ГБРС КПП</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>541</width>
            <height>81</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_6">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>10</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>3711013676</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>140</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Лог</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>111</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Запрос!</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Цена контракта с</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>70</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>70</y>
            <width>171</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>200000000000</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>70</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>до</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
